portal.go

```diff
@@ -1394,6 +1415,8 @@ func (portal *Portal) handleIMAttachment(msg *imessage.Message, attach *imessage
 		}
 	}
 
+	extraContent["protocol"] = portal.buildProtocol(msg.Service)
```

Comment:
The key should be `fi.mau.imessage.protocol` (and it should probably be a `const` somewhere else since it's used in more than one place)

Reply:
Actually `fi.mau.imessage.service` to match the `m.bridge` data

Reply:
Oh wait, it's putting a lot of stuff there. Just the service would probably be enough